addon/mixins/sortable-item.js

```diff
@@ -119,6 +119,16 @@ export default Mixin.create({
    * @public
    */
   maxScrollSpeed: 20,
+  
+  /**
+   * How many pixes the element needs to be moved to be considered dragged
+   *
+   * @property draggingThreshold
+   * @type Number
+   * @default 15
+   * @public
+   */
+  draggingThreshold: 15,
```

Comment:
@lukemelia I guess it's hard to make a click without moving the mouse even by 1 pixels. I played with different values and that one came out as a good compromise.

In my option the value should block accidentally dragging by clicking for most of the situations  by default.